### ISSUE TITLE
Facebook Page Plugin legacy widget: Timeline does not load in Chrome

### ISSUE THREAD

**Issue by jp-imagines** [Automattic/jetpack#34842]
### Impacted plugin

Jetpack

### Quick summary

The legacy "Facebook Page Plugin" widget from Jetpack has an option to display a Facebook page's timeline. This seems to work in Firefox and Safari, but the timeline never loads in Chrome.

Seeing the same console error as mentioned in https://github.com/Automattic/jetpack/issues/20881, though that issue says the error didn't affect the widget's functioning at that time.

### Steps to reproduce

1. Install, activate, and connect Jetpack. (Or start with a WordPress.com site.)
2. Navigate to Jetpack > Dashboard > Settings > Writing, and ensure `Make extra widget available for use on your site including subscription forms and more` is enabled.
3. Activate a non-block theme (i.e. any theme that supports widgets).
4. Go to Appearance > Widgets and add the Facebook Page Plugin (Jetpack) widget. Enter a Facebook page URL and enable the "show timeline" option.

### A clear and concise description of what you expected to happen.

The Facebook timeline should display on the live site in the widget.

### What actually happened

In Chrome, the timeline never loads and shows an infinite loading icon instead. Browser console shows:

```
ErrorUtils caught an error:

Cannot listen to an undefined element. [Caught in: Tried to listen to element of type click]

Subsequent non-fatal errors won't be logged; see https://fburl.com/debugjs.
```

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Atomic

### Logs or notes

Only tested on WoA so far, but I assume this affects self-hosted as well. Unsure if Simple sites are affected.

Timeline loads properly in Firefox and Safari, but not Chrome.

**Comment by jp-imagines** [comment]
Initial report: 7502917-zd-a8c

**Comment by bobmatyas** [comment]
@jp-imagines I tested this, but the FB widget fails to load in Chrome, Firefox, and Safari. However, I see a different error in the console:

![Annotation on 2024-01-03 at 17-50-09](https://github.com/Automattic/jetpack/assets/45246438/738a644c-bde8-468a-a5b1-048de7a97288)

You were able to view this in other browsers without issue? Did you try this on a test site? I tested on a self-hosted test site and had the same results:

![Annotation on 2024-01-03 at 18-00-11](https://github.com/Automattic/jetpack/assets/45246438/234ffcf8-b542-4fa5-85f1-3b894ff35d70)

With more information, we can keep investigating.

**Comment by jp-imagines** [comment]
> You were able to view this in other browsers without issue? Did you try this on a test site? I tested on a self-hosted test site and had the same results:

Tested on an Atomic test site with that user's Facebook profile URL, and it is working for me in Firefox and Safari still.

Weirdly, the timeline _is_ loading without issue in an incognito Chrome window, and the same error from my initial report appears, so that definitely doesn't seem to be related. Being logged into/out of Facebook (or WordPress.com) doesn't seem to make a difference, and disabling browser extensions doesn't seem to affect anything in my tests either. 

I did just notice another browser console error though (which isn't present when the timeline loads in incognito), though it's different than yours @bobmatyas:

![facebook error](https://github.com/Automattic/jetpack/assets/5630717/4171097d-4cf4-463e-8913-57438cd3a83c)

**Comment by jeherve** [comment]
I cannot seem to be able to reproduce on my end, in Firefox and in Chrome, whether I'm in Incognito / Private mode or logged in with a Facebook account.

Do you happen to use privacy / tracker browser extensions in your browser?

**Comment by jp-imagines** [comment]
I was still replicating the issue even with all browser extensions disabled. I cleared cookies for Facebook in Chrome again today though, and that has fixed it – the timeline is now loading without issue. I thought I did that last week, but I must not have.

The user who originally reported this has since disabled the timeline on their widget as a workaround, but I'll let them know to clear their Facebook cookies again to see if that fixes things properly for them too. 

I'll close out this issue now.
